src/commands/interactive.js

```
@@ -1,571 +1,567 @@
         return;
-      }
-      
-      // Handle standard keys
-      if (key === '\u0003') { // Ctrl+C
-        process.exit(0);
-      } else if (key === '\r' || key === '\n') { // Enter
-        if (inputValue.trim() === 'exit' || inputValue.trim() === 'quit') {
-          process.exit(0);
+
+      case 'groups':
+        const groupNames = Object.keys(groups);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)